docs/source/howto/_plot_x_y_databroker.ipynb

```diff
@@ -563,22 +590,123 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
      },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "xyplot(dataset, \"m1\", \"noisy\", title=\"Example XY plot\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d2052f64",
+   "metadata": {},
+   "source": [
+    "Alternatively, the dataset can be referenced directly from the databroker catalog object `cat`.  (Here, `cat[-1]` is the `run`.)\n",
+    "\n",
+    "Since the `x` and `y` objects (defined above) have the data variable names, the names can be obtained directly from the respective axis object.\n",
```

Comment:
This is super confusing. How do I know that x and y, who were defined based on a different run (86), will still be the same (m1 and noisy) for the -1 run? And what do you mean by "data variable names"?
Calling them "axis object" is also confusing. Are you referring to matplotlib axis object (I don't think so) .
I do understand what you are doing from the code, but not really from the explanations.

Reply:
Since the extra example introduces confusion, perhaps it can be removed?  

The term _data variable_ was used above.  It is the term used by `xarray.Dataset` in Section 3.

![Clipboard01](https://user-images.githubusercontent.com/2279984/236947352-f50d0a52-c24b-406a-a456-233dd5468309.png)

The term _axis object_ could be changed to something more obvious.  Do you have a suggestion?  This refers to the Python object (not the text name), `dataset["m1"]` or `dataset["noisy"]`

Reply:
What if we provide a link from [data variable](https://docs.xarray.dev/en/stable/generated/xarray.Dataset.html) to the xarray documentation (as shown here)?

Reply:
How about this text in Step 4?

> Pick the `m1` (motor readback value) and `noisy` data for $x$ and $y$, respectively.  We'll refer to these as _data variables_ to be consistent with the `xarray.Dataset` shown above.  In that table, the data for `m1`, `m1_user_setpoint` and `noisy` are listed as [data variables](https://docs.xarray.dev/en/stable/generated/xarray.Dataset.html).